engine/runtime/src/test/scala/org/enso/interpreter/test/instrument/RuntimeServerTest.scala

```diff
@@ -457,46 +647,121 @@ class RuntimeServerTest
               )
             ),
             Api.SuggestionsDatabaseUpdate.Add(
-              Suggestion.Local(
-                Some(idResult),
+              Suggestion.Method(
+                None,
                 moduleName,
-                "result",
+                "bar",
+                Seq(Suggestion.Argument("this", "Any", false, false, None)),
+                "here",
                 "Any",
-                Suggestion.Scope(
-                  Suggestion.Position(0, 6),
-                  Suggestion.Position(2, 21)
-                )
+                None
               )
             )
           )
         )
       )
     )
-    context.consumeOut shouldEqual List("1337")
+  }
 
-    // Modify the file
+  it should "send updates from multi liner" in {
```

Comment:
name this better, `multi liner` sounds strangely gangster for a thing existing in a programming language :D